plonky2/src/lookup_test.rs

```diff
@@ -1,28 +1,34 @@
-static LOGGER_INITIALIZED: Once = Once::new();
```

Comment:
This file has a lot of changes, basically just noise related to removing duplicate exports from within individual tests to the module level. Only the feature gating of `LOGGER_INITIALIZED` matters here.